javascript/cable_ready.js

```diff
@@ -235,6 +235,15 @@ const DOMOperations = {
     dispatch(element, 'cable-ready:after-set-style', detail)
   },
 
+  setStyles: detail => {
+    const { element, styles } = detail
+    for (const property in styles) {
+      const name = property,
+        value = styles[property]
+      DOMOperations['setStyle']({ element: element, name: name, value: value })
+    }
+  },
+
```

Comment:
Should we dispatch before/after events for this?

Reply:
Won't those events be generated by the setStyle operations this creates?

Reply:
Unless you see value in highlighting that several values were updated in addition to the events generated by the individual updates. I feel like it's a bit much, but I'm listening!

Reply:
It's definitely a lot of events, but it would be consistent. Also, wondering if we should suppress the individual events when `setSetyles` is used and only emit for the batch operation. I'm curious to see what @excid3 thinks.

Reply:
Great questions. I feel like it'd be good to be consistent and have events for `setStyles`. 

We could also skip the delegation to `setStyle` if firing those individual events is a problem. 

I don't personally have a use for those events, so I'm not sure I've got the best opinion on those.